chart/epinio/templates/server.yaml

```diff
@@ -238,7 +238,7 @@ spec:
             - name: REGISTRY_CERTIFICATE_SECRET
               value: "{{ .Values.registry.certificateSecret }}"
             {{- end }}
-          image: splatform/epinio-server:{{ .Chart.AppVersion }}
+          image: "{{ trimSuffix "/" .Values.registryURL }}/{{ .Values.image.epinio.repository}}:{{ or .Values.image.epinio.tag .Chart.AppVersion }}"
```

Comment:
Does that mean that if registryURL is set to "" image will be 

```
    image: /splatform/epinio:v0.2.0
```
Can it start with a "/"

Reply:
I didn't think of that. I was trying to make it work both with and without a `/` on the `registryURL`. Maybe it needs a helper after all...

Reply:
Maybe: 

```
 {{ if registryURL }}{{ trimSuffix "/" .Values.registryURL }}/{{ end }}{{ .Values.image.epinio.repository}}
```

Reply:
Yep I now see the value of the helper

Reply:
ok re-introduced the helper. Hopefully it's fine now.